src/app/teacher/edit-tag/edit-tag.component.html

```diff
@@ -12,4 +13,9 @@
     <mat-label i18n>Color</mat-label>
     <input matInput (keyup.enter)="enterKeyPressed()" [formControl]="colorControl" />
   </mat-form-field>
+  <button mat-button (click)="close()" i18n>Cancel</button>
+  <button mat-raised-button color="primary" [disabled]="!nameControl.valid" (click)="submit()" i18n>
```

Comment:
Do we need the i18n tag here?
```suggestion
  <button mat-raised-button color="primary" [disabled]="!nameControl.valid" (click)="submit()">
```